src/deploy/functions/release/reporter.ts

```diff
@@ -56,30 +57,83 @@ export class AbortedDeploymentError extends DeploymentError {
 }
 
 /** Add debugger logs and GA metrics for deploy stats. */
-export async function logAndTrackDeployStats(summary: Summary): Promise<void> {
+export async function logAndTrackDeployStats(
+  summary: Summary,
+  context?: args.Context
+): Promise<void> {
   let totalTime = 0;
   let totalErrors = 0;
   let totalSuccesses = 0;
   let totalAborts = 0;
   const reports: Array<Promise<void>> = [];
 
   const regions = new Set<string>();
+  const codebases = new Set<string>();
   for (const result of summary.results) {
+    const fnDeployEvent = {
+      platform: result.endpoint.platform,
+      trigger_type: backend.endpointTriggerType(result.endpoint),
+      region: result.endpoint.region,
+      runtime: result.endpoint.runtime,
+      status: !result.error
+        ? "success"
+        : result.error instanceof AbortedDeploymentError
+        ? "aborted"
+        : "failure",
+      duration: result.durationMs,
+    };
+    reports.push(trackGA4("function_deploy", fnDeployEvent));
+
     const tag = triggerTag(result.endpoint);
     regions.add(result.endpoint.region);
+    codebases.add(result.endpoint.codebase || "default");
     totalTime += result.durationMs;
     if (!result.error) {
       totalSuccesses++;
+      if (
+        context?.codebaseDeployEvents &&
+        context.codebaseDeployEvents[result.endpoint.codebase || "default"] !== undefined
+      ) {
+        context.codebaseDeployEvents[result.endpoint.codebase || "default"]
+          .fn_deploy_num_successes++;
+      }
       reports.push(track("function_deploy_success", tag, result.durationMs));
```

Comment:
Could you remove these calls to track? They'll stop collecting data in a few days, and  getting rid of them now make it a little easier to merge old & new metrics in BQ